components/ReviewCard.tsx

```diff
@@ -85,6 +86,7 @@ export const ReviewCard: React.FC<ReviewCardProps> = ({ submissionData }) => {
     await review({
       variables: {
         submissionId: id,
+        lessonId: lessonId,
```

Comment:
Can this be destructured? From `lessonId: lessonId,` to `'lessonId,`

Reply:
Done!